charts/jaeger-operator/crds/crd.yaml

```
@@ -2,10 +2,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   name: jaegers.jaegertracing.io
-  annotations:
-    cert-manager.io/inject-ca-from: {{ default {{ .Release.Namespace }} .Values.certs.certificate.namespace }}/{{ default "jaeger-operator-service-cert" .Values.certs.certificate.secretName }}
   labels:
-{{ include "jaeger-operator.labels" . | indent 4 }}
+    app: jaeger-operator
```

How about following the generic label convenction as in [here](https://github.com/jaegertracing/helm-charts/blob/main/charts/jaeger-operator/templates/_helpers.tpl#L45)

---

I can absolutely add these, I had just reverted to what was set in the past I beleive. Since we can't make use of helm templating here does it make sense to just make both labels `jaeger-operator`?
```suggestion
    app.kubernetes.io/name: jaeger-operator
    app.kubernetes.io/instance: jaeger-operator
```